zkevm-circuits/src/state_circuit.rs

```diff
@@ -56,9 +59,9 @@ pub struct StateCircuitConfig<F> {
     // For Rw::AccountStorage, identify non-existing if both committed value and
     // new value are zero. Will do lookup for ProofType::StorageDoesNotExist if
     // non-existing, otherwise do lookup for ProofType::StorageChanged.
-    // TODO: use BatchedIsZeroGadget here, once it doesn't depend on the evm circuit constraint
-    // builder.
-    is_non_exist: Column<Advice>,
+    is_non_exist: BatchedIsZeroConfig,
+    // Intermediarey witness used to reduce mpt lookup expression degree
```

Comment:
Intermediarey -> Intermediary

Reply:
Tanks!  Applied in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1046/commits/8c84bc31ed1ffc74c31508b5818e90d8762dd450